src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -10,10 +10,9 @@ class BreadcrumbsHelper extends CoreBreadcrumbsHelper
      */
     protected $_defaultConfig = [
         'templates' => [
-            'wrapper' => '<ol{{attrs}}>{{content}}</ol>',
-            'item' => '<li{{attrs}}><a href="{{url}}"{{innerAttrs}}>{{title}}</a></li>{{separator}}',
-            'itemWithoutLink' => '<li{{attrs}}><span{{innerAttrs}}>{{title}}</span></li>{{separator}}',
-            'separator' => '<li{{attrs}}><span{{innerAttrs}}>{{separator}}</span></li>'
```

Comment:
The core Breadcrumbs helper requires the `separator` template. So please add it back with empty string as template value.

Reply:
Fixed with commit 69b9a53